dev/devicelab/lib/tasks/integration_tests.dart

```diff
@@ -59,6 +61,22 @@ TaskFunction createAndroidSemanticsIntegrationTest() {
   );
 }
 
+TaskFunction createFlutterCreateOfflineTest() {
+  return () async {
+    final Directory tempDir = Directory.systemTemp.createTempSync('flutter_create_test.');
+    String output;
+    await inDirectory(tempDir, () async {
+      output = await eval(path.join(flutterDirectory.path, 'bin', 'flutter'), <String>['create', '--offline', 'flutter_create_test']);
+    });
+    if (output.contains(RegExp('building flutter tool', caseSensitive: false))) {
```

Comment:
My reasoning is that the steps in the [beta release process](https://github.com/flutter/flutter/wiki/Release-process) describe checking out an arbitrary tag and verifying that `flutter create --offline` doesn't rebuild anything on it. The tag refs are just regular commits like anything else, as far as I can tell. I don't see a reason they would be different from non-tagged refs.

I may be totally missing something here though, I'm unfortunately not seeing a way to totally verify these tests work until after they're merged.

I can't run any of the devicelab tests on a completely clean checkout:

```
$ dart bin/run.dart -t smoke_test_success
Error: Could not resolve the package 'args' in 'package:args/args.dart'.
Error: Could not resolve the package 'path' in 'package:path/path.dart'.
Error: Could not resolve the package 'flutter_devicelab' in 'package:flutter_devicelab/framework/manifest.dart'.
Error: Could not resolve the package 'flutter_devicelab' in 'package:flutter_devicelab/framework/runner.dart'.
Error: Could not resolve the package 'flutter_devicelab' in 'package:flutter_devicelab/framework/utils.dart'
```

After running `flutter update packages` in the checkout this passes.

Reply:
On a clean checkout, `flutter update-packages` builds the flutter tool. That's why it doesn't get rebuild when you run the tests and they pass.

I guess, it's ok then because you can't run the tests without running `flutter update-packages`.